### PR TITLE
Reversing post order

### PR DESCRIPTION
**Pull request by Edward-Phillips**
posts are now displayed in reverse chronological order

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def create
   end
 
   def index
-    @posts = Post.all
+    @posts = Post.all.reverse
```
Comment (FayeCarter) [inline app/controllers/posts_controller.rb:12]: concise 👍🏻

File: config/application.rb [hunk 2]
```diff
@@ -11,6 +11,9 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
 
+    # Use GMT
+    config.time_zone = 'Europe/London'
```
Comment (FayeCarter) [inline config/application.rb:15]: Great location

**Comment by FayeCarter** [comment]
LGTM!

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (tommyrharper) [inline app/controllers/posts_controller.rb:12]: Love the simplicity of this

File: spec/features/user_can_submit_posts_spec.rb [hunk 3]
```diff
@@ -1,11 +1,16 @@
 require 'rails_helper'
 
 RSpec.feature "Timeline", type: :feature do
-  scenario "Can submit posts and view them" do
```
Comment (tommyrharper) [inline spec/features/user_can_submit_posts_spec.rb:4]: Good testing :)

**Pull request merged**